modules/rules_synology/0.2.0/presubmit.yml

```diff
@@ -0,0 +1,15 @@
+matrix:
+  platform:
+    - ubuntu2204
+    - macos
+  bazel: [7.x]
```

Comment:
```suggestion
  bazel: [7.x, 8.x]
```

Reply:
Thanks for suggestions — I’ll patch into this rev